### PR TITLE
Enable the installation of specific versions with the "install_kustomize.sh" script (#3022)

### PR DESCRIPTION
**Pull request by tiwalter**
Enables the feature #3022

### CODE REVIEW

**Comment by tiwalter** [comment]
I signed it

File: hack/install_kustomize.sh [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
+# kustomize binary to your current working directory.
+# (e.g. 'install_kustomize.sh')
+#
+# If an argument is given -> Downloads the specified version of the
+# kustomize binary to your current working directory.
+# (e.g. 'install_kustomize.sh 3.8.2')
 #
 # Fails if the file already exists.
 
+if [ -z "$1" ]
```
Comment (monopole) [inline hack/install_kustomize.sh:13]: Please stick to the formatting of other if/then statements.

File: hack/install_kustomize.sh [hunk 2]
```diff
@@ -36,11 +50,17 @@ curl -s https://api.github.com/repos/kubernetes-sigs/kustomize/releases |\
   grep browser_download |\
   grep $opsys |\
   cut -d '"' -f 4 |\
-  grep /kustomize/v |\
+  grep /kustomize/v$version |\
   sort | tail -n 1 |\
   xargs curl -s -O -L
 
-tar xzf ./kustomize_v*_${opsys}_amd64.tar.gz
+if [ -e ./kustomize_v*_${opsys}_amd64.tar.gz ]
```
Comment (monopole) [inline hack/install_kustomize.sh:57]: same formatting please

File: hack/install_kustomize.sh [hunk 3]
```diff
@@ -1,10 +1,24 @@
...
+#
+# If an argument is given -> Downloads the specified version of the
+# kustomize binary to your current working directory.
+# (e.g. 'install_kustomize.sh 3.8.2')
 #
 # Fails if the file already exists.
 
+if [ -z "$1" ]
+  then
+    echo "No version specified. Downloading the most recently released kustomize binary."
```
Comment (monopole) [inline hack/install_kustomize.sh:15]: No echo statements except on error.
Other scripts use this, and expect no output on success (as with almost all unix commands).

**Review by monopole — COMMENTED** [review]
thanks for this contribution

File: hack/install_kustomize.sh (hunk 1 above)
Comment (tiwalter) [reply hack/install_kustomize.sh]: the formatting is now correct.

File: hack/install_kustomize.sh (hunk 2 above)
Comment (tiwalter) [reply hack/install_kustomize.sh]: also correct now

File: hack/install_kustomize.sh (hunk 3 above)
Comment (tiwalter) [reply hack/install_kustomize.sh]: I removed the unnecessary echo statements.

**Review by monopole — APPROVED** [review]
/lgtm
thanks!

**Pull request merged**